.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
Where are you using gitpod?

Reply:
I could not run the application locally, so I had to use Gitpod to finish my work on time. Let me know if that works @sumana2001

Reply:
I think it's ok if you don't commit the file and add it to .gitignore